testsuite/python/utils.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+import unittest as ut
+import espressomd.utils
+
+
+class UtilsTest(ut.TestCase):
```

Comment:
this file should be in `CMakeLists.txt` :-)

Reply:
ah right